public/index.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/pixlcrypt_mono.png">
```

Comment:
Specify image type for png? https://stackoverflow.com/questions/6121725/favicon-ico-or-png-correct-tags

Reply:
ok!